app/javascript/packs/Notification/notification.jsx

```diff
@@ -227,7 +228,8 @@ export default class Notification extends React.Component {
                             </Row>                                                                          
                         </Modal.Body>
                     <Modal.Footer>
-                        <Col>Komentarze</Col>
+                        <Col><p>Komentarze</p>
+                        <MessageByNotification notifID={this.props.NotificationID} /></Col>
```

Comment:
prefer using full names of variables/props
```suggestion
                        <MessageByNotification notificationID={this.props.NotificationID} /></Col>
```